### DISCUSSION TITLE
Proposal: Enable Sponsorship

### DISCUSSION THREAD

**Discussion by JacobCoffee** [litestar-org/litestar#1098] [Ideas]
**What's the feature you'd like to ask for.**
Enable sponsorship for the Starlite project. We could use the funds to advertise, enhance our branding (name, logo, etc), pay content creators to create videos, etc.

**Additional context**
![image](https://user-images.githubusercontent.com/45884264/214437651-c937394e-fd92-46e9-8b28-6b6e912e7b70.png)

**Comment by Goldziher** [comment]
Well, the reason I didnt do this is that I wanted to make it clear that Starlite was a not for profit endevour. But, having sponsors might be helpful, I dont know. Lets see what the others think.

**Reply by jarmos-san** [reply]
Personal pockets can only be so deep, so I don't think there's any issue with asking for sponsorship. On top of it, you can be transparent with where the finances are utilised & stay non-profit as well.

End of the day, don't worry, the project will be community effort & sponsorship from the community itself will help move the project further in many other ways.

**Reply by JacobCoffee** [reply]
>  On top of it, you can be transparent with where the finances are utilised & stay non-profit as well.

https://opencollective.com/

It is really nice. It shows who paid what, and what we paid for (full I/O transparency) 

examples:
https://opencollective.com/sanic-org
https://opencollective.com/vuejs

**Comment by JacobCoffee** [comment]
https://opencollective.com/

**Comment by JacobCoffee** [comment]
This is mostly done, but on hold for now until the time is right.
